features/polyExplorer/src/screens/dataExploration/dataExploration.jsx

```diff
@@ -422,11 +408,9 @@ const DataExplorationScreen = () => {
                         highlight={highlights[entity.ppid]?.dataRecipient}
                         maxCompanies={maxCompanies}
                     />
-                    <p className="bubble-source">
```

Comment:
Keep in mind that, since you have updated the "source" CSS styles in the `InfoButton` component, now you will also need to update the CSS styles of those "sources" without an InfoButton (for instance, `bubble-source` is still used in some screens in `DataExploration.jsx`).